### PR TITLE
speed-dreams, bump version, fix libpng16 reference

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
WIP, still need to fix some credits in the patchset, just posting so maybe some could have a look :)

File: games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+   HINTS ENV PLIB_DIR
+   PATH_SUFFIXES 
+-	Headers include/plib include
++	Headers include/plib include develop/headers develop/headers/plib
+   PATHS
+ #Additional MacOS Paths
+-   	 ~/Library/Frameworks/plib.framework
+-  	 /Library/Frameworks/plib.framework
+- 	 /System/Library/Frameworks/plib.framework # Tiger
++	 ~/Library/Frameworks/plib.framework
```
Comment (korli) [inline games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset]: you could probably avoid to change these lines.

File: games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+ 
+ IF(UNIX)
+     SD_INSTALL_FILES(MAN man6 PREFIX ${SOURCE_DIR}/doc/man FILES ${_TARGET_NAME}.6)
+@@ -36,4 +36,4 @@ IF((APPLE) AND ("${CMAKE_INSTALL_PREFIX}" MATCHES "\\.app$"))
+    INCLUDE(BundleUtilities)
+    SET(BU_CHMOD_BUNDLE_ITEMS ON)
+    FIXUP_BUNDLE(\"${APPS}\" \"${PlugIns}\" \"${DIRS}\")")
+-endif()
+\ No newline at end of file
++endif()
```
Comment (korli) [inline games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset]: you could remove this chunk

File: games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset (hunk 1 above)
Comment (Begasus) [reply games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset]: Haven't check yet, but thought so too (looking at the file), leftover from previous patch (as the next comment) :)

File: games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset (hunk 2 above)
Comment (Begasus) [reply games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset]: @korli this is generated by extracting the patchset?

Comment (korli) [reply games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset]: not exactly, you edited with Pe which added this newline at the end. this is just noise in this patch and won't be upstreamed.

Comment (Begasus) [reply games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset]: Ok, need to investigate that how to remove without braking the patchset

Comment (Begasus) [reply games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset]: Think this should do it

**Comment by Begasus** [comment]
All architectures disabled, game crashes on launch, not sure it's related to the game or SDL2/GL

**Comment by Begasus** [comment]
Fixed the build for the previous version, but that one also fails to start, so disabled it too

**Comment by Begasus** [comment]
Related to lib:libSDL*_1.2 https://github.com/haikuports/haikuports/issues/3878

**Pull request merged**